nixos/modules/services/security/oauth2-proxy-nginx.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, ... }:
+with lib;
+let
+  cfg = config.services.oauth2-proxy.nginx;
+in
+{
+  options.services.oauth2-proxy.nginx = {
+    proxy = mkOption {
+      type = types.str;
+      default = config.services.oauth2-proxy.httpAddress;
+      defaultText = literalExpression "config.services.oauth2-proxy.httpAddress";
+      description = lib.mdDoc ''
```

Comment:
\> `with lib`
\> `lib.mdDoc`

This is why I don't like `with`.

Reply:
That's from a treewide change. I don't think it really matters tbh.

Reply:
Yes, but many "new code" repeats this nonsense.

Indeed in the short run it does not matter.